xml/depl_maintenance.xml

```diff
@@ -2180,5 +2180,22 @@ migrate</screen>
       </para>
      </step>
   </procedure>
- </sect1>
+ </section>
+ <xi:include href="operations-maintenance-database_maintenance.xml"/>
+ <section xml:id="sec-depl-maintenance-fernet-tokens">
+  <title>Rotating Fernet Tokens</title>
+  <para>
+     Fernet tokens should be rotated frequently for security purposes.
+     It is recommended to setup this task as a cron job in
+     <literal>/etc/cron.weekly/openstack-keystone-fernet</literal>
+     on the server where the keystone service is running,
```

Comment:
Instead of "on the server where the keystone service is running", how about this?

"on the keystone server designated as a master node in a highly available setup, with the following content:"